src/lib/learn-client/api/progress/get-tutorial-progress.ts

```diff
@@ -0,0 +1,48 @@
+import { get, toError } from 'lib/learn-client'
+import { Collection, Tutorial } from 'lib/learn-client/types'
+import { SessionData } from 'types/auth'
+import { ApiCollectionTutorialProgress } from '../api-types'
+import { getTutorialProgressRoute } from './util'
+
+interface GetTutorialProgressOptions {
+	accessToken: SessionData['accessToken']
+	tutorialId: Tutorial['id']
+	collectionId: Collection['id']
+}
+
+type GetTutorialProgressResult = null | ApiCollectionTutorialProgress
+
+/**
+ * Fetches progress on a single tutorial.
+ *
+ * https://digital-api-specs.vercel.app/learn#tag/Collection-Tutorial-Progress/paths/~1collections~1{collection_id}~1tutorials~1{tutorial_id}~1progress/get
+ */
+const getTutorialProgress = async ({
```

Comment:
I've added get, update, and create functions for `lib/learn-client/progress`. I expect we'll rarely if ever use these directly, seems more likely we'll always use them through the related `react-query` hooks.